languages/swift/exercises/concept/dictionaries/.docs/introduction.md

```diff
@@ -62,5 +62,17 @@ for (name, address) in addresses {
 // Buffy Summers lives at 1630 Revello Drive
 ```
 
-[dictionaries]: https://docs.swift.org/swift-book/LanguageGuide/CollectionTypes.html#ID113
-[dictionary-docs]: https://developer.apple.com/documentation/swift/Dictionary
+Like the other collection types, dictionaries can be sorted according to arbitrary sorting functions. To do so, one must first define a function that takes two _key: value_ pairs, represented as tuples and returns a `Bool` such that the return value is `true` if the first _key: value_ pair should appear in the sorted result _before_ the second _key: value_ pair.
```

Comment:
I appreciate that additional sorting section and it's well understandable for me.